client/src/Components/CountryCard/Card.js

```diff
@@ -22,6 +24,8 @@ const Card = () => {
         }}
       </AppContextConsumer>
       <FriendList />
+    <CardHeader />
```

Comment:
I think CardHeader and BorderBay should go above FriendList (that's how David had it before).
Also make sure you fix the formatting.